.pre-commit-config.yaml

```diff
@@ -43,3 +43,7 @@ repos:
         entry: python -m scripts.generate_pip_deps_from_conda
         files: ^(environment.yml|requirements-dev.txt)$
         pass_filenames: false
+-   repo: https://github.com/MarcoGorelli/flake8-rst-mirror
+    rev: v0.7.0
+    hooks:
+    -   id: flake8-rst
```

Comment:
Will this use the settings from setup.cfg?

Reply:
@dsaxton yup - if I apply this
```diff
diff --git a/setup.cfg b/setup.cfg
index d938d2ef3..8b26424e4 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -34,7 +34,6 @@ exclude =
 
 [flake8-rst]
 bootstrap =
-    import numpy as np
     import pandas as pd
     np  # avoiding error when importing again numpy or pandas
     pd  # (in some cases we want to do it to show users)
```
then the hook will fail